### PR TITLE
Do not nuke session on unknown backend errors

### PR DESCRIPTION
**Pull request by estrattonbailey**
Relies on https://github.com/bluesky-social/atproto/pull/2010

### CODE REVIEW

File: src/state/session/index.tsx [hunk 1]
```diff
@@ -102,10 +102,21 @@ function createPersistSessionHandler(
...
+  } = {},
 ): AtpPersistSessionHandler {
   return function persistSession(event, session) {
     const expired = event === 'expired' || event === 'create-failed'
 
+    if (event === 'network-error') {
+      logger.warn(`session: persistSessionHandler received network-error event`)
+      unstable_networkErrorCallback?.()
+      return
+    }
```
Comment (estrattonbailey) [inline src/state/session/index.tsx:118]: Only real addition is this block, just kicks the user out to signup and sends a warning to sentry

Comment (pfrazee) [reply src/state/session/index.tsx:118]: Why in a sub-object with the unstable_ prefix?

Comment (estrattonbailey) [reply src/state/session/index.tsx]: Oh, just removed, was just what I called it initially

**Review by pfrazee — APPROVED** [review]
Was able to successfully test on web by blocking `getSession`. Good step in the right direction, since this makes it so you don't need to input your password.

**Pull request merged**